src/tests.rs

```diff
@@ -33,16 +33,16 @@ fn sell_price_should_work() {
 }
 
 #[test]
-fn buy_price_should_work() {
+fn in_given_out_should_work() {
     let cases = vec![
-        (1000, 2000, 500, Some(334), "Easy case"),
+        (2000, 1000, 500, Some(334), "Easy case"),
```

Comment:
Due to the change of order of params, this has changed. But I wonder if there is good reason to do as it would required more effort than just renaming the func where it is currently used.

Reply:
I implemented it as mentioned in the issue, there the parameters were
switched. I wanted to point this out but forgot.

I can order it the way you think it makes sense.

Reply:
no.all good